src/schema/v2/bulkUpdatePartnerArtworksMutation.ts

```diff
@@ -124,7 +113,19 @@ export const updatePartnerArtworksMutation = mutationWithClientMutationId<
     }
 
     try {
-      return await updatePartnerArtworksLoader(id, gravityOptions)
+      const gravityResponse = await updatePartnerArtworksLoader(
+        id,
+        gravityOptions
+      )
+
+      // In the future it could be helpful to have a list of successfully opted in ids, can add this to gravity at a later date
+
+      const formattedReturn = {
+        updatedPartnerArtworks: { count: gravityResponse.success, ids: [] },
+        skippedPartnerArtworks: gravityResponse.errors,
```

Comment:
@lilyfromseattle maybe let's be explicit here so future devs don't have to go digging into the gravity response format
```
skippedPartnerArtworks: { count: gravityResponse.errors.count, ids: gravityResponse.errors.ids }
```